data/campaigns/The_South_Guard/scenarios/02_Proven_by_the_Sword.cfg

```diff
@@ -483,6 +472,115 @@
         [/if]
     [/event]
 
+    # you can get some mermen reinforcements if you go on the village here
+    # it will replay the event from the first scenario if you didn't get them there
+
+    [event]
+        name=moveto
+        [filter]
+            side=1
+            x,y=14,20
+        [/filter]
+
+        [if]
+            [variable]
+                name=has_mermen
+                equals="yes"
+            [/variable]
+            [then]
+                [unit]
+                    side=1
+                    type=Merman Fighter
+                    id=Absu
+                    name= _ "Absu"
+                    [modifications]
+                        {TRAIT_LOYAL}
+                        {TRAIT_STRONG}
+                    [/modifications]
+                    x,y=14,21
+                    {IS_LOYAL}
+                [/unit]
+
+                #po : deplorable means awful or terrible
+                [message]
+                    speaker=Absu
+                    message= _ "Freedom! Sir Knight, please allow me to join you in fighting off these deplorable thugs."
```

Comment:
Not sure about the reasoning – I would expect that it does not matter much for them if loyalists or outlaws life around. As long as they can life their life…

Sounds more like this makes clear to the player that the Outlaws are the Always Bad Guys, and I think we don't need an other, to the actual story unrelated, reasoning for them to be the bad ones.

Reply:
I've changed the text here; perhaps it's reasonable now?